server/tsuserver.py

```diff
@@ -105,24 +112,58 @@ def get_player_count(self):
         return len(self.client_manager.clients)
 
     def load_config(self):
-        with open('config/config.yaml', 'r') as cfg:
+        with open('config/config.yaml', 'r', encoding = 'utf-8') as cfg:
             self.config = yaml.load(cfg)
+            self.config['motd'] = self.config['motd'].replace('\\n', ' \n') 
 
     def load_characters(self):
-        with open('config/characters.yaml', 'r') as chars:
+        with open('config/characters.yaml', 'r', encoding = 'utf-8') as chars:
             self.char_list = yaml.load(chars)
         self.build_char_pages_ao1()
 
     def load_music(self):
-        with open('config/music.yaml', 'r') as music:
+        with open('config/music.yaml', 'r', encoding = 'utf-8') as music:
             self.music_list = yaml.load(music)
         self.build_music_pages_ao1()
         self.build_music_list_ao2()
         
-
+    def load_ids(self):
+        self.ipid_list = {}
+        self.hdid_list = {}
+        #load ipids
+        try:
+            with open('storage/ip_ids.json', 'r', encoding = 'utf-8') as whole_list:
+                self.ipid_list = json.loads(whole_list.read())
+        except:
+            pass
```

Comment:
hm, should i just check if 'ip_ids.json' is exist? 🤔 I believe that after first launch this error will never appear again.